npm_and_yarn/lib/dependabot/npm_and_yarn/file_fetcher.rb

```diff
@@ -566,6 +566,12 @@ def ignore_package_lock?
         npmrc.content.match?(/^package-lock\s*=\s*false/)
       end
 
+      def ignore_pnpm_lock?
```

Comment:
The word `ignore` confused me for a sec, as I thought we somehow supported that in https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#ignore

Looks like this is instead actually trying to respect this setting: https://pnpm.io/npmrc#lockfile

Maybe call it `skip_pnpm_lock?` to avoid confusion?

I'm also fine as-is...

Reply:
I just copied it from the analogous `npm` setting, but I can rename both.

Reply:
Renamed and aligned analogous method for NPM at #7197.